### PR TITLE
AF 1957 fixes

### PR DESCRIPTION
**Pull request by ederign**
Fixing system config produces FS to uses GIT as scheme, fixing some pom deps and also deploy files created automatically.

Also undo the reverting in the gabriel's commit.

@pefernan is that everything right?

### CODE REVIEW

**Comment by ederign** [comment]
Let's wait for the green full downstream build that time ;)

File: uberfire-backend/uberfire-backend-cdi/src/main/java/org/uberfire/backend/server/cdi/SystemConfigProducer.java [hunk 1]
```diff
@@ -407,6 +405,12 @@ public void destroy(final FileSystem instance,
         };
     }
 
+    URI resolveFSURI(SpacesAPI spaces, Space space, String fsName) {
+        return spaces.resolveFileSystemURI(SpacesAPI.Scheme.GIT,
```
Comment (ederign) [inline uberfire-backend/uberfire-backend-cdi/src/main/java/org/uberfire/backend/server/cdi/SystemConfigProducer.java]: https://github.com/kiegroup/appformer/blob/a55967db7656ade01a11b4f190968d2f06e3dc30/uberfire-backend/uberfire-backend-cdi/src/main/java/org/uberfire/backend/server/cdi/SystemConfigProducer.java#L293-L305

Comment (ederign) [reply uberfire-backend/uberfire-backend-cdi/src/main/java/org/uberfire/backend/server/cdi/SystemConfigProducer.java]: Returning original behaviour.

**Comment by barboras7** [comment]
jenkins execute full downstream build

**Comment by ederign** [comment]
@barboras7 this is basically the same PR that you reviewd with a small fix.

**Comment by ederign** [comment]
jenkins retest this

**Comment by barboras7** [comment]
@ederign Thanks Eder, I know. Just wanted to check if it builds correctly as recently there were some troubles with full downstream build

**Comment by ederign** [comment]
@mbiarnes -> /home/jenkins/workspace/KIE/master/pullrequest/appformer-compile-downstream-build/downstream-repos/kie-wb-distributions/business-central-parent/business-central-webapp/target/business-central-webapp/WEB-INF/lib/wildfly-elytron-1.1.0.Final.jar: No space left on device -> [Help 1]

**Comment by ederign** [comment]
Please jenkins execute a full downstream build

**Comment by domhanak** [comment]
jenkins execute full downstream build

**Comment by ederign** [comment]
@pefernan could you please recheck this one?

**Comment by ederign** [comment]
@barboras7 could you please do a quick sanity check on this one? Is the old one + the git fix.

**Comment by barboras7** [comment]
Hi @gabriel376,
I was working with the feature and I found this scenario. I am not sure what I am doing wrong but I am getting results different from what I am expecting. 

See the following steps: 

1. In a business central without the feature, create a dataset1 and page1 (having a chart based on the dataset1). Add page1 into the workbench menu (Design - Projects, Pages, page1)
2. Shut down the business central.
3. Move the .niogit folder to some other location. 
4. Into standalone/deployments, put a business-central.war with the feature and run the business central
5. Create a dataset2 and page2 (having a chart based on the dataset2). Add page2 into the workbench menu (Design - Projects, Pages, page2)
6. Shut down the business central
7. Merge the new .niogit folder with the old one
8. Start the business central
9. Check the datasets in the UI (only the dataset2 is visible)

When checking the datasets in the .niogit folder, I can see both datasets (dataset1 and dataset2) although not named 'dataset1' and 'dataset2'. Also, I do not see the page1 in the perspectives.git in niogit.

**Pull request merged**